.codeclimate.yml

```diff
@@ -15,7 +13,12 @@ ratings:
   paths:
   - "src/**/*.js"
 exclude_paths:
-- dist/**/*
```

Comment:
I think it's safer to keep `dist/` since this folder is pushed [in tags](/chartjs/Chart.js/tree/v2.6.0/dist) - in case we want CC score for releases.